api/v2beta1/helmrelease_types.go

```diff
@@ -246,15 +245,6 @@ type HelmChartTemplateSpec struct {
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
 
-	// Determines what enables the creation of a new artifact. Valid values are
-	// ('ChartVersion', 'Revision').
-	// See the documentation of the values for an explanation on their behavior.
-	// Defaults to ChartVersion when omitted.
-	// +kubebuilder:validation:Enum=ChartVersion;Revision
-	// +kubebuilder:default:=ChartVersion
-	// +optional
-	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
```

Comment:
My guess is that the answer is yes.

Reply:
Missed this comment for some reason :-S

No, it was introduced to the `HelmChart` resource in a later version than the `reconcilers-dev` branch. So you don't have access to the API against this version, but it will return.

Reply:
Ok good, will keep this open and loop back when it becomes relevant.